### ISSUE TITLE
The Advanced Settings Aren't Searchable

### ISSUE THREAD

**Issue by scrutinizer11** [iina/iina#4512]
**System and IINA version:**

- macOS 10.14
- IINA 1.3.2

**Expected behavior:**

Search results for the advanced settings show up in the pop-up

**Actual behavior:**

No entries, although the settings exist and no typos being made.

**Steps to reproduce:**

1. Open IINA's settings window.
2. In the search field type any part of the name of a setting from the Advanced pane (i.e, "logging" to search for _Extra logging_)
3. The search returns an empty result. 

**How often does this happen?**

Always.

**Comment by low-batt** [comment]
Reproduced. Thank you for reporting this.

**Comment by low-batt** [comment]
### Analysis

The title isn't quite correct as some of the `Advanced` settings are found in a search. _However_ the specific example given, "logging", is indeed missing.

The search index is generated dynamically by the class `PreferenceWindowController`. The first phase of index generation loads the NIBs for the settings panel tabs. The NIBs are then scaned to pick out search strings. Adding debug logging that prints out the results of the scan shows this for the `Advanced` tab:
```text
14:55:30.647 [iina][d] Advanced
14:55:30.647 [iina][d]   Settings
14:55:30.647 [iina][d]     Use mpv's OSD
14:55:30.647 [iina][d]     Additional mpv options
14:55:30.647 [iina][d]     Use config directory:
```

That is all the scan found. All the logging controls are missing as well as the `Enable advanced settings` button.

The reason why the scan is having trouble is that it depends upon certain conventions being followed by the settings UI. The scan requires that each settings tab be divided into subview sections and that each section contains a label whose identifier starts with "SectionTitle". If such a label can not be found in the subview then the scan ignores that subview. Two subviews in the `Advanced` tab are missing this special label.

Another requirement of the scan is that buttons that should be included in a search have an identifier that starts with "FunctionalButton". Some buttons on the `Advanced` and `Utilities` tabs were being ignored because they were missing such an identifier.

### Fixing

- Add `SectionTitleTop` hidden label to `Advanced` tab
- Add `SectionTitleLogging` hidden label to `Advanced` tab
- Add `FunctionalButtonShowLog` identifier to `Show log viewer` button on  `Advanced` tab
- Add `FunctionalButtonLogDirectory`  identifier to `Open log directory` button on  `Advanced` tab
- Add `FunctionalButtonShowConfig` identifier to `Show the config file in Finder` button on `Key Bindings` tab
- Add `FunctionalButtonImportConfig` identifier to `Import an existing config file` button on `Key Bindings` tab
- Add `FunctionalButtonDefaultApp` identifier to `Set IINA as the Default Application…` button on `Utilities` tab
- Add `formSearchTerm` method to `PreferenceWindowController` that removes extraneous characters from labels and titles
- Correct the spelling of available on `General` tab
- Change `Use media keys` to be  `Use system media control` to match UI
- Add a `logLabelDict` method to `PreferenceWindowController` that logs the search terms found by scanning NIBs

Before the changes:

<details>
<summary><b>IINA 1.3.2 NIB scan result:</b></summary>

```text
14:55:30.647 [iina][d] ---------------------------------------------------
14:55:30.647 [iina][d] Search text found in scan of preference panel NIBs:
14:55:30.647 [iina][d] Advanced
14:55:30.647 [iina][d]   Settings
14:55:30.647 [iina][d]     Use mpv's OSD
14:55:30.647 [iina][d]     Additional mpv options
14:55:30.647 [iina][d]     Use config directory:
14:55:30.647 [iina][d] Key Bindings
14:55:30.647 [iina][d]   Settings
14:55:30.647 [iina][d]     Use media keys
14:55:30.647 [iina][d]     Configuration:
14:55:30.647 [iina][d]     Display raw values
14:55:30.647 [iina][d] Video/Audio
14:55:30.647 [iina][d]   Video:
14:55:30.647 [iina][d]     Number of threads:
14:55:30.647 [iina][d]     Hardware decoder:
14:55:30.647 [iina][d]     Force dedicated GPU
14:55:30.647 [iina][d]     Load ICC profile
14:55:30.647 [iina][d]     Enable HDR mode
14:55:30.647 [iina][d]     Enable tone mapping
14:55:30.647 [iina][d]     Target peak:
14:55:30.647 [iina][d]     nits
14:55:30.647 [iina][d]     Algorithm:
14:55:30.647 [iina][d]   Audio:
14:55:30.647 [iina][d]     Preferred language:
14:55:30.647 [iina][d]     S/PDIF output:
14:55:30.648 [iina][d]     AC3
14:55:30.648 [iina][d]     DTS
14:55:30.648 [iina][d]     DTS-HD
14:55:30.648 [iina][d]     Initial volume:
14:55:30.648 [iina][d]     Maximum volume:
14:55:30.648 [iina][d]     Preferred audio device:
14:55:30.648 [iina][d]     Number of threads:
14:55:30.648 [iina][d] General
14:55:30.648 [iina][d]   History:
14:55:30.648 [iina][d]     Enable playback history
14:55:30.648 [iina][d]     Enable "Open Recent" menu
14:55:30.648 [iina][d]     Track all played files in "Open Recent" menu
14:55:30.648 [iina][d]   Behavior:
14:55:30.649 [iina][d]     At launch:
14:55:30.649 [iina][d]     Always open media in new window
14:55:30.649 [iina][d]     Quit after all windows are closed
14:55:30.649 [iina][d]     Keep window open after playback finishes
14:55:30.649 [iina][d]     Resume last playback position
14:55:30.649 [iina][d]     Check for updates
14:55:30.649 [iina][d]     Use legacy full screen
14:55:30.649 [iina][d]     Black out other monitors while in full screen
14:55:30.650 [iina][d]     Receive beta updates
14:55:30.650 [iina][d]     Switch to "music mode" automatically when playing an audio file
14:55:30.650 [iina][d]     When media is opened:
14:55:30.650 [iina][d]     Pause
14:55:30.650 [iina][d]     Enter fullscreen
14:55:30.650 [iina][d]     Pause/resume when:
14:55:30.650 [iina][d]     Minimized/un-minimized
14:55:30.650 [iina][d]     Window becomes inactive/active
14:55:30.650 [iina][d]     Play upon entering full screen
14:55:30.650 [iina][d]     Pause upon leaving full screen
14:55:30.650 [iina][d]     Pause when machine goes to sleep
14:55:30.650 [iina][d]   Screenshots:
14:55:30.650 [iina][d]     Format:
14:55:30.650 [iina][d]     Include subtitles
14:55:30.650 [iina][d]     Save to
14:55:30.650 [iina][d]     Copy to clipboard
14:55:30.650 [iina][d]     Show previews after taking screenshots
14:55:30.650 [iina][d]   Playlist:
14:55:30.650 [iina][d]     Add files in the same folder automatically
14:55:30.650 [iina][d]     Play next item automatically
14:55:30.650 [iina][d]     Show artist and track name for audio files when avaliable
14:55:30.650 [iina][d]     Only in "music mode"
14:55:30.650 [iina][d] Subtitle
14:55:30.650 [iina][d]   Auto Load:
14:55:30.650 [iina][d]     Advanced
14:55:30.650 [iina][d]     Subtitles have priority when filename containing:
14:55:30.650 [iina][d]     Also search subtitles in following directories:
14:55:30.650 [iina][d]   Text Subtitles:
14:55:30.650 [iina][d]     Font:
14:55:30.651 [iina][d]     Size:
14:55:30.651 [iina][d]     Color:
14:55:30.651 [iina][d]     Italic
14:55:30.651 [iina][d]     Background:
14:55:30.651 [iina][d]     Bold
14:55:30.651 [iina][d]     Size:
14:55:30.651 [iina][d]     Color:
14:55:30.651 [iina][d]     Advanced
14:55:30.651 [iina][d]     Offset:
14:55:30.651 [iina][d]     Color:
14:55:30.651 [iina][d]     Blur:
14:55:30.651 [iina][d]     Font spacing:
14:55:30.651 [iina][d]   Position:
14:55:30.651 [iina][d]      X:
14:55:30.651 [iina][d]     Y:
14:55:30.651 [iina][d]     Vertical position:
14:55:30.651 [iina][d]     Display subtitles in letterboxes while in full screen
14:55:30.651 [iina][d]     Scale subtitles with window size
14:55:30.651 [iina][d]     Y:
14:55:30.651 [iina][d]      X:
14:55:30.651 [iina][d]   Online Subtitles:
14:55:30.651 [iina][d]     Download subtitles from:
14:55:30.651 [iina][d]     OpenSubtitles account:
14:55:30.651 [iina][d]     Assrt API token:
14:55:30.651 [iina][d]     Search online subtitles automatically
14:55:30.651 [iina][d]   ASS Subtitles:
14:55:30.651 [iina][d]     Ignore ASS styles
14:55:30.651 [iina][d]     Override level:
14:55:30.651 [iina][d]     strip
14:55:30.651 [iina][d]   Other:
14:55:30.651 [iina][d]     Preferred language:
14:55:30.651 [iina][d]     Default encoding:
14:55:30.651 [iina][d] UI
14:55:30.651 [iina][d]   Appearance:
14:55:30.651 [iina][d]     Theme:
14:55:30.651 [iina][d]   On Screen Controller:
14:55:30.651 [iina][d]     Auto hide after:
14:55:30.651 [iina][d]     Snap to center when dragging
14:55:30.651 [iina][d]     Show chapter position in progress bar
14:55:30.651 [iina][d]     Show remaining time instead of total duration
14:55:30.651 [iina][d]     Layout:
14:55:30.651 [iina][d]     Use left/right button for:
14:55:30.651 [iina][d]     Toolbar:
14:55:30.651 [iina][d]   Thumbnail Preview:
14:55:30.651 [iina][d]     Enable thumbnail preview
14:55:30.651 [iina][d]     MB
14:55:30.652 [iina][d]     Maximum cache size:
14:55:30.652 [iina][d]     Width:
14:55:30.652 [iina][d]     px
14:55:30.652 [iina][d]   Window:
14:55:30.652 [iina][d]     Use physical resolution on Retina displays
14:55:30.652 [iina][d]     Resize the window to fit video size:
14:55:30.652 [iina][d]     When media is opened manually
14:55:30.652 [iina][d]     Disabled
14:55:30.652 [iina][d]     Resize window to:
14:55:30.652 [iina][d]     Always
14:55:30.652 [iina][d]     Always float on top while playing
14:55:30.652 [iina][d]     Initial window size:
14:55:30.652 [iina][d]     Initial window position:
14:55:30.652 [iina][d]     Always show float on top status in the title bar
14:55:30.652 [iina][d]   Picture-in-
Picture:
14:55:30.652 [iina][d]     Toggle Picture-in-Picture by minimizing/un-minimizing the window
14:55:30.652 [iina][d]     When entering Picture-in-Picture:
14:55:30.652 [iina][d]     Do nothing
14:55:30.652 [iina][d]     Hide
14:55:30.652 [iina][d]     Minimize
14:55:30.652 [iina][d]     Automatically pause playback
14:55:30.652 [iina][d]   On Screen Display:
14:55:30.652 [iina][d]     Auto hide after:
14:55:30.652 [iina][d]     Display time and battery info when in full screen
14:55:30.652 [iina][d]     Text size:
14:55:30.652 [iina][d]     Enable OSD
14:55:30.652 [iina][d] Network
14:55:30.652 [iina][d]   youtube-dl:
14:55:30.652 [iina][d]     Enable youtube-dl
14:55:30.652 [iina][d]     Raw options:
14:55:30.652 [iina][d]     Custom youtube-dl path:
14:55:30.652 [iina][d]   Cache:
14:55:30.652 [iina][d]     Enable cache
14:55:30.652 [iina][d]     Default cache size (KB):
14:55:30.652 [iina][d]     Seconds to prefetch:
14:55:30.652 [iina][d]   Network:
14:55:30.652 [iina][d]     User agent:
14:55:30.652 [iina][d]     Transport RTSP stream through:
14:55:30.652 [iina][d]     HTTP proxy:
14:55:30.652 [iina][d] Control
14:55:30.652 [iina][d]   Trackpad
14:55:30.652 [iina][d]     Pinch to:
14:55:30.652 [iina][d]     Force Touch to:
14:55:30.652 [iina][d]   Mouse
14:55:30.652 [iina][d]     Seek type:
14:55:30.652 [iina][d]     Scroll horizontally to:
14:55:30.652 [iina][d]     Sensitivity for normal seek:
14:55:30.652 [iina][d]     Single click to:
14:55:30.653 [iina][d]     Double click to:
14:55:30.653 [iina][d]     Right click to:
14:55:30.653 [iina][d]     Scroll vertically to:
14:55:30.653 [iina][d]     Sensitivity for volume:
14:55:30.653 [iina][d]     Middle click to:
14:55:30.653 [iina][d]     Accepts first mouse click when not focused
14:55:30.653 [iina][d] Utilities
14:55:30.653 [iina][d]   Default Application
14:55:30.653 [iina][d]   Get Browser Extensions for IINA
14:55:30.653 [iina][d]     Chrome
14:55:30.653 [iina][d]     Firefox
14:55:30.653 [iina][d]   Clear Cache
14:55:30.653 [iina][d]     Clear Saved Playback Progress…
14:55:30.653 [iina][d]     Clear Playback History…
14:55:30.653 [iina][d]     Clear Thumbnail Cache…
14:55:30.653 [iina][d]     Current thumbnail cache:
14:55:30.653 [iina][d]   Restore Alerts
14:55:30.653 [iina][d]     Restore Suppressed Alerts…
14:55:30.653 [iina][d] ---------------------------------------------------
```
</details>

With the changes in the PR applied:
<details>
<summary><b>Corrected NIB scan result:</b></summary>

```text
22:00:53.723 [iina][d] --------------------------------------------------
22:00:53.723 [iina][d] Search terms found in scan of settings panel NIBs:
22:00:53.723 [iina][d] UI
22:00:53.723 [iina][d]   Thumbnail Preview
22:00:53.723 [iina][d]     Enable thumbnail preview
22:00:53.723 [iina][d]     MB
22:00:53.723 [iina][d]     Maximum cache size
22:00:53.723 [iina][d]     Width
22:00:53.723 [iina][d]     px
22:00:53.723 [iina][d]   Appearance
22:00:53.723 [iina][d]     Theme
22:00:53.723 [iina][d]   On Screen Controller
22:00:53.723 [iina][d]     Auto hide after
22:00:53.723 [iina][d]     Snap to center when dragging
22:00:53.723 [iina][d]     Show chapter position in progress bar
22:00:53.723 [iina][d]     Show remaining time instead of total duration
22:00:53.723 [iina][d]     Layout
22:00:53.723 [iina][d]     Use left/right button for
22:00:53.723 [iina][d]     Toolbar
22:00:53.723 [iina][d]   Window
22:00:53.723 [iina][d]     Use physical resolution on Retina displays
22:00:53.723 [iina][d]     Resize the window to fit video size
22:00:53.723 [iina][d]     When media is opened manually
22:00:53.723 [iina][d]     Disabled
22:00:53.723 [iina][d]     Resize window to
22:00:53.723 [iina][d]     Always
22:00:53.723 [iina][d]     Always float on top while playing
22:00:53.723 [iina][d]     Initial window size
22:00:53.723 [iina][d]     Initial window position
22:00:53.723 [iina][d]     Always show float on top status in the title bar
22:00:53.723 [iina][d]   On Screen Display
22:00:53.723 [iina][d]     Auto hide after
22:00:53.723 [iina][d]     Display time and battery info when in full screen
22:00:53.723 [iina][d]     Text size
22:00:53.723 [iina][d]     Enable OSD
22:00:53.723 [iina][d]   Picture-in-Picture
22:00:53.723 [iina][d]     Toggle Picture-in-Picture by minimizing/un-minimizing the window
22:00:53.723 [iina][d]     When entering Picture-in-Picture
22:00:53.723 [iina][d]     Do nothing
22:00:53.723 [iina][d]     Hide
22:00:53.723 [iina][d]     Minimize
22:00:53.723 [iina][d]     Automatically pause playback
22:00:53.727 [iina][d] Video/Audio
22:00:53.727 [iina][d]   Video
22:00:53.727 [iina][d]     Number of threads
22:00:53.727 [iina][d]     Hardware decoder
22:00:53.727 [iina][d]     Force dedicated GPU
22:00:53.727 [iina][d]     Load ICC profile
22:00:53.727 [iina][d]     Enable HDR mode
22:00:53.727 [iina][d]     Enable tone mapping
22:00:53.727 [iina][d]     Target peak
22:00:53.727 [iina][d]     nits
22:00:53.727 [iina][d]     Algorithm
22:00:53.727 [iina][d]   Audio
22:00:53.727 [iina][d]     Preferred language
22:00:53.727 [iina][d]     S/PDIF output
22:00:53.727 [iina][d]     AC3
22:00:53.727 [iina][d]     DTS
22:00:53.727 [iina][d]     DTS-HD
22:00:53.727 [iina][d]     Initial volume
22:00:53.727 [iina][d]     Maximum volume
22:00:53.727 [iina][d]     Preferred audio device
22:00:53.727 [iina][d]     Number of threads
22:00:53.727 [iina][d] General
22:00:53.727 [iina][d]   Playlist
22:00:53.728 [iina][d]     Add files in the same folder automatically
22:00:53.728 [iina][d]     Play next item automatically
22:00:53.728 [iina][d]     Show artist and track name for audio files when available
22:00:53.728 [iina][d]     Only in music mode
22:00:53.728 [iina][d]   Screenshots
22:00:53.728 [iina][d]     Format
22:00:53.728 [iina][d]     Include subtitles
22:00:53.728 [iina][d]     Save to
22:00:53.728 [iina][d]     Copy to clipboard
22:00:53.728 [iina][d]     Show previews after taking screenshots
22:00:53.728 [iina][d]   History
22:00:53.728 [iina][d]     Enable playback history
22:00:53.728 [iina][d]     Enable Open Recent menu
22:00:53.728 [iina][d]     Track all played files in Open Recent menu
22:00:53.728 [iina][d]   Behavior
22:00:53.728 [iina][d]     At launch
22:00:53.728 [iina][d]     Always open media in new window
22:00:53.728 [iina][d]     Quit after all windows are closed
22:00:53.728 [iina][d]     Keep window open after playback finishes
22:00:53.728 [iina][d]     Resume last playback position
22:00:53.728 [iina][d]     Check for updates
22:00:53.728 [iina][d]     Use legacy full screen
22:00:53.728 [iina][d]     Black out other monitors while in full screen
22:00:53.728 [iina][d]     Receive beta updates
22:00:53.728 [iina][d]     Switch to music mode automatically when playing an audio file
22:00:53.728 [iina][d]     When media is opened
22:00:53.728 [iina][d]     Pause
22:00:53.728 [iina][d]     Enter fullscreen
22:00:53.728 [iina][d]     Pause/resume when
22:00:53.728 [iina][d]     Minimized/un-minimized
22:00:53.728 [iina][d]     Window becomes inactive/active
22:00:53.728 [iina][d]     Play upon entering full screen
22:00:53.728 [iina][d]     Pause upon leaving full screen
22:00:53.728 [iina][d]     Pause when machine goes to sleep
22:00:53.728 [iina][d] Subtitle
22:00:53.728 [iina][d]   Other
22:00:53.728 [iina][d]     Preferred language
22:00:53.728 [iina][d]     Default encoding
22:00:53.728 [iina][d]   Auto Load
22:00:53.728 [iina][d]     Advanced
22:00:53.728 [iina][d]     Subtitles have priority when filename containing
22:00:53.728 [iina][d]     Also search subtitles in following directories
22:00:53.728 [iina][d]   ASS Subtitles
22:00:53.728 [iina][d]     Ignore ASS styles
22:00:53.728 [iina][d]     Override level
22:00:53.728 [iina][d]     strip
22:00:53.728 [iina][d]   Text Subtitles
22:00:53.728 [iina][d]     Font
22:00:53.728 [iina][d]     Size
22:00:53.728 [iina][d]     Color
22:00:53.728 [iina][d]     Italic
22:00:53.728 [iina][d]     Background
22:00:53.728 [iina][d]     Bold
22:00:53.728 [iina][d]     Size
22:00:53.728 [iina][d]     Color
22:00:53.728 [iina][d]     Advanced
22:00:53.728 [iina][d]     Offset
22:00:53.728 [iina][d]     Color
22:00:53.728 [iina][d]     Blur
22:00:53.728 [iina][d]     Font spacing
22:00:53.728 [iina][d]   Online Subtitles
22:00:53.728 [iina][d]     Download subtitles from
22:00:53.728 [iina][d]     OpenSubtitles account
22:00:53.728 [iina][d]     Assrt API token
22:00:53.728 [iina][d]     Search online subtitles automatically
22:00:53.728 [iina][d]   Position
22:00:53.728 [iina][d]     X
22:00:53.728 [iina][d]     Y
22:00:53.728 [iina][d]     Vertical position
22:00:53.728 [iina][d]     Display subtitles in letterboxes while in full screen
22:00:53.728 [iina][d]     Scale subtitles with window size
22:00:53.728 [iina][d]     Y
22:00:53.728 [iina][d]     X
22:00:53.728 [iina][d] Control
22:00:53.728 [iina][d]   Mouse
22:00:53.728 [iina][d]     Seek type
22:00:53.728 [iina][d]     Scroll horizontally to
22:00:53.728 [iina][d]     Sensitivity for normal seek
22:00:53.728 [iina][d]     Single click to
22:00:53.728 [iina][d]     Double click to
22:00:53.728 [iina][d]     Right click to
22:00:53.728 [iina][d]     Scroll vertically to
22:00:53.728 [iina][d]     Sensitivity for volume
22:00:53.729 [iina][d]     Middle click to
22:00:53.729 [iina][d]     Accepts first mouse click when not focused
22:00:53.729 [iina][d]   Trackpad
22:00:53.729 [iina][d]     Pinch to
22:00:53.729 [iina][d]     Force Touch to
22:00:53.729 [iina][d] Key Bindings
22:00:53.729 [iina][d]   Settings
22:00:53.729 [iina][d]     Use system media control
22:00:53.729 [iina][d]     Configuration
22:00:53.729 [iina][d]     Show the config file in Finder
22:00:53.729 [iina][d]     Display raw values
22:00:53.729 [iina][d]     Import an existing config file
22:00:53.729 [iina][d] Network
22:00:53.729 [iina][d]   Network
22:00:53.729 [iina][d]     User agent
22:00:53.729 [iina][d]     Transport RTSP stream through
22:00:53.729 [iina][d]     HTTP proxy
22:00:53.729 [iina][d]   Cache
22:00:53.729 [iina][d]     Enable cache
22:00:53.729 [iina][d]     Default cache size KB
22:00:53.729 [iina][d]     Seconds to prefetch
22:00:53.729 [iina][d]   youtube-dl
22:00:53.729 [iina][d]     Enable youtube-dl
22:00:53.729 [iina][d]     Raw options
22:00:53.729 [iina][d]     Custom youtube-dl path
22:00:53.729 [iina][d] Utilities
22:00:53.729 [iina][d]   Default Application
22:00:53.729 [iina][d]     Set IINA as the Default Application
22:00:53.729 [iina][d]   Get Browser Extensions for IINA
22:00:53.729 [iina][d]     Chrome
22:00:53.729 [iina][d]     Firefox
22:00:53.729 [iina][d]   Restore Alerts
22:00:53.729 [iina][d]     Restore Suppressed Alerts
22:00:53.729 [iina][d]   Clear Cache
22:00:53.729 [iina][d]     Clear Saved Playback Progress
22:00:53.729 [iina][d]     Clear Playback History
22:00:53.729 [iina][d]     Clear Thumbnail Cache
22:00:53.729 [iina][d]     Current thumbnail cache
22:00:53.729 [iina][d] Advanced
22:00:53.729 [iina][d]   Top
22:00:53.729 [iina][d]     Enable advanced settings
22:00:53.729 [iina][d]   Settings
22:00:53.729 [iina][d]     Use mpv's OSD
22:00:53.729 [iina][d]     Additional mpv options
22:00:53.729 [iina][d]     Use config directory
22:00:53.729 [iina][d]   Logging
22:00:53.729 [iina][d]     Open log directory
22:00:53.729 [iina][d]     Show log viewer
22:00:53.729 [iina][d]     Log level
22:00:53.729 [iina][d]     Enable logging to file
22:00:53.729 [iina][d] --------------------------------------------------
```
</details>

Searching with the fix applied:

![issue-4512](https://github.com/iina/iina/assets/86170219/71da41b2-a6f7-4b9c-a16c-2c3ed9fa114b)

**Comment by low-batt** [comment]
The fix for this issue has been merged into the IINA `develop` branch. GitHub [automatically closed](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) the linked issue in reaction to the merge. I am reopening this issue until the fix is available in an official release of IINA so that users intending to report this problem can easily locate this existing report. Once the fix for this issue has been included in an official release this issue will be closed.

**Comment by low-batt** [comment]
This was fixed in IINA 1.3.5.
